auth/src/test/java/com/firebase/ui/auth/ui/idp/AuthMethodPickerActivityTest.java

```diff
@@ -146,18 +146,16 @@ public void testFacebookLoginFlow() {
         assertNotNull(facebookButton);
         facebookButton.performClick();
 
-        ShadowActivity.IntentForResult nextIntent =
-                Shadows.shadowOf(authMethodPickerActivity).getNextStartedActivityForResult();
-        verifySaveCredentialIntent(nextIntent, FacebookAuthProvider.PROVIDER_ID);
+        Intent smartLockIntent = SmartLock.getInstance(authMethodPickerActivity).getIntentForTest();
```

Comment:
I'll try that again, but I keep getting a null pointer exception at line 221 in smartlock.java.